src/main/java/gov/usgs/aqcu/retrieval/RatingCurveListService.java

```
@@ -0,0 +1,56 @@
+package gov.usgs.aqcu.retrieval;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import java.time.Instant;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.aquaticinformatics.aquarius.sdk.timeseries.AquariusClient;
+import com.aquaticinformatics.aquarius.sdk.timeseries.servicemodels.Publish.RatingCurveListServiceRequest;
+import com.aquaticinformatics.aquarius.sdk.timeseries.servicemodels.Publish.RatingCurveListServiceResponse;
+
+import net.servicestack.client.IReturn;
+import net.servicestack.client.WebServiceException;
+
+@Component
+public class RatingCurveListService {
+	private static final Logger LOG = LoggerFactory.getLogger(RatingCurveListService.class);
+
+	@Autowired
+	private String aquariusUrl;
+
+	@Autowired
+	private String aquariusUser;
+
+	@Autowired
+	private String aquariusPassword;
+
+	public RatingCurveListServiceResponse get(String ratingModelIdentifier, Double utcOffset, Instant startDate, Instant endDate) {
+		RatingCurveListServiceRequest request = new RatingCurveListServiceRequest()
+				.setRatingModelIdentifier(ratingModelIdentifier)
+				.setUtcOffset(utcOffset)
+				.setQueryFrom(startDate)
+				.setQueryTo(endDate);
+		RatingCurveListServiceResponse ratingCurveResponse = executePublishApiRequest(request);
+		return ratingCurveResponse;
+	}
+
+	public <TResponse> TResponse executePublishApiRequest(IReturn<TResponse> request) {
+		try (AquariusClient client = AquariusClient.createConnectedClient(aquariusUrl, aquariusUser, aquariusPassword)) {
+			return client.Publish.get(request);
+		} catch (WebServiceException e) {
+			LOG.error("A web service error occurred while fetching data from Aquarius: \nStatus: " + e.getStatusCode() + "\nError Code: "
```

Should we make these LOG.error messages more explicit about which service failed (in this case, "A web service error occurred while fetching Rating Curve data...")?  Could help pinpoint where an api change may have broken things.

---

Yeah, I think that's a good idea. I think refactoring this a little bit so that this same function isn't duplicated in each retrieval service is probably worth doing. @dsteinich Mentioned that this should really re-throw an exception after logging and then the caller can handle the exception and log with a more specific error message.